com.borgbase.Vorta.yaml

```diff
@@ -0,0 +1,94 @@
+app-id: com.borgbase.Vorta
+
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+branch: stable
+
+command: vorta
+
+finish-args:
+  # X11/Wayland
+  - --socket=x11
+  - --share=ipc
+  - --socket=wayland
+  - --device=dri
+  # Sync
+  - --share=network
+  # Filesystem access
+  - --filesystem=home
+  - --filesystem=/run/media
+  - --filesystem=/media
```

Comment:
Just merge all of these into `--filesystem=host`

Reply:
Done